src/main/java/com/kruczala/michal/AdvanceCalculator.java

```diff
@@ -1,32 +1,74 @@
 package com.kruczala.michal;
 
-import com.kruczala.michal.exceptions.DivideByZeroIsWrongException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+
+import com.kruczala.michal.exceptions.ApplicationException;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
 public class AdvanceCalculator {
+    private static final Logger LOGGER = LoggerFactory.getLogger(AdvanceCalculator.class);
+
     public double multiply(double firstNumber, int secondNumber) {
-        return firstNumber * secondNumber;
+        double result = firstNumber * secondNumber;
+        LOGGER.debug("first number: {} second number: {} result of multiply : {}", firstNumber, secondNumber, result);
+        return result;
     }
 
     public double multiply(double firstNumber, double secondNumber) {
+        double result = firstNumber * secondNumber;
         if (firstNumber == 8 && secondNumber == Float.MAX_VALUE) {
-            return 1500100900;
+            LOGGER.debug("first number: " + firstNumber + "second number:" + secondNumber + "result:" + result);
```

Comment:
To się zaloguje jeżeli program wejdzie do if, a rezultat jest zwracany zawsze. Logowanie trzeba przenieść niżej

Reply:
chyba zrobione :)